tests/test_ota_proxy/test_ota_proxy_server.py

```diff
@@ -237,11 +237,28 @@ async def ota_image_downloader(self, regular_entries, sync_event: asyncio.Event)
                 url = urljoin(
                     cfg.OTA_IMAGE_URL, quote(f'/data/{entry.path.relative_to("/")}')
                 )
-                async with session.get(url, proxy=self.OTA_PROXY_URL) as resp:
-                    hash_f = sha256()
-                    async for data, _ in resp.content.iter_chunks():
-                        hash_f.update(data)
-                    assert hash_f.hexdigest() == entry.sha256hash
+                _retry_count_for_exceed_hard_limit = 0
+                # NOTE: for space_availability==exceed_hard_limit,
+                #       it is normal that transition is interrupted when
+                #       space_availability transfered from below_hard_limit to exceed_hard_limit.
```

Comment:
Let me confirm, this `space_availability transfered from below_hard_limit to exceed_hard_limit.` means, 
space_availability is changed in the test?
I think space_availability is set by the fixture params so the value seems NOT to be changed.